language/stdlib/modules/Libra.move

```diff
@@ -311,12 +296,17 @@ module Libra {
         preburn_address: address
     ): Libra<CoinType> acquires BurnCapability, CurrencyInfo, Preburn {
         let addr = Signer::address_of(account);
-        assert(exists<BurnCapability<CoinType>>(addr), Errors::not_published(EBURN_CAPABILITY));
+        assert(exists<BurnCapability<CoinType>>(addr), Errors::requires_capability(EBURN_CAPABILITY));
         cancel_burn_with_capability(
             preburn_address,
             borrow_global<BurnCapability<CoinType>>(addr)
         )
     }
+    spec fun cancel_burn {
+        pragma aborts_if_is_partial = true;
+        aborts_if !exists<BurnCapability<CoinType>>(Signer::spec_address_of(account)) with Errors::REQUIRES_CAPABILITY; // Aborts if no BurnCap [B12]
+        aborts_with Errors::NOT_PUBLISHED, Errors::LIMIT_EXCEEDED;
```

Comment:
You needed this because of the partial, right? Wondering whether it was easy to figure this out. This is hardly documented.